From 1d444772db14f4acb8d47b874462b0cc171ab672 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 17 Feb 2025 12:01:59 -0800
Subject: [PATCH] libmpathutils/uxsock.c: Include string.h for memcpy

Fixes
uxsock.c:72:2: error: call to undeclared library function 'memcpy' with type 'void *(void *, const void *, unsigned long)'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]

Upstream-Status: Submitted [https://lore.kernel.org/dm-devel/20250217200503.4166910-1-raj.khem@gmail.com/T/#u]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
Cc: Benjamin Marzinski <bmarzins@redhat.com>
Cc: Martin Wilck <mwilck@suse.com>
---
 libmpathutil/uxsock.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libmpathutil/uxsock.c b/libmpathutil/uxsock.c
index 2135476d..a474874e 100644
--- a/libmpathutil/uxsock.c
+++ b/libmpathutil/uxsock.c
@@ -8,6 +8,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <stdarg.h>
+#include <string.h>
 #include <fcntl.h>
 #include <sys/ioctl.h>
 #include <sys/types.h>
